subsys/shell/shell_rtt.c

```diff
@@ -9,6 +9,10 @@
 #include <SEGGER_RTT.h>
 #include <logging/log.h>
 
+BUILD_ASSERT_MSG(!(IS_ENABLED(CONFIG_LOG_BACKEND_RTT) &&
```

Comment:
I was told once by @carlescufi  we should use either: `__ASSERT` or `__ASSERT_NO_MSG` macros

Reply:
but this is static assert.

Reply:
sorry, missed that !